doc/user/external.xml

```diff
@@ -377,6 +399,12 @@ conda install -c conda-forge ninja
             implement those actions via shell commands in the &ninja; build file.
         </para>
 
+        <para>
+            When ninja runs the generated &ninja; build file, ninja will launch &scons; as a daemon and feed commands
+            to that &scons; process which &ninja; is unable to build directly.  This daemon will stay alive until
+            explicitly killed or it times out. The timeout is set by &cv-link-NINJA_SCONS_DAEMON_KEEP_ALIVE; .
```

Comment:
Maybe add a sentence here: "The daemon will be restarted if the SConscripts change or the build changes in a way that ninja determines it needs to regenerate the build.ninja file"

Reply:
Done.